dynamiq/nodes/node.py

```diff
@@ -217,6 +217,7 @@ class Node(BaseModel, Runnable, ABC):
     _output_references: NodeOutputReferences = PrivateAttr()
 
     model_config = ConfigDict(arbitrary_types_allowed=True)
+    _input_schema: ClassVar[type[BaseModel] | None] = None
```

Comment:
It can't be changed or overridden on instance creation
If you are meaning changing on class level, the both could be changed if user would like to break something.

Reply:
```
from pydantic import BaseModel, Field

class B(BaseModel):
    b: str = Field(default="bee", frozen=True, description="Bee is b", title="B")

class A(BaseModel):
    a: ClassVar[B] = B
    _a: ClassVar[B] = B

a = A(a="test", _a="test")
```
```
>>> print(a)

>>> a.a
<class '__main__.B'>
>>> a._a
<class '__main__.B'>
```